### PR TITLE
5.0.0 readme updates

### PR DESCRIPTION
**Pull request by alexandern17**
Updated Troubleshooting Guide readme

### CODE REVIEW

File: TROUBLESHOOTING-GUIDE.md [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
 
-The Thunderhead SDK for Android Troubleshooting Guide for Common Implementation Issues.
+The Thunderhead SDK for Android Troubleshooting Guide for common implementation issues.
 
-## Integration Issues
+## Table of Contents
+
+- [Integration issues](#integration-issues)
+  * [How to resolve StackOverflow Exception](#how-to-resolve-stackoverflow-exception)
+  * [Resolving conflicts with `android:allowBackup`](#resolving-conflicts-with-android-allowbackup)
```
Comment (tatianasavinykh) [inline TROUBLESHOOTING-GUIDE.md]: @alexandern17 The link does not work

File: TROUBLESHOOTING-GUIDE.md [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+  * [Resolving > The library com.google.firebase:firebase-iid is being requested by various other libraries](#resolving-the-library-comgooglefirebase-firebase-iid-is-being-requested-by-various-other-libraries)
```
Comment (tatianasavinykh) [inline TROUBLESHOOTING-GUIDE.md]: @alexandern17 The link does not work. Also not sure about special char in the headers/TOC

File: TROUBLESHOOTING-GUIDE.md [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+  * [Resolving: NoSuchMethodError for Base64 class or 15_000: Signpost cannot be used on this platform](#resolving-nosuchmethoderror-for-base64-class-or-15-000-signpost-cannot-be-used-on-this-platform)
+- [Performance issues](#performance-issues)
+  * [Build Time](#build-time)
+- [Error codes and resolutions](#error-codes-and-resolutions)
+  * [14019: Non Adaptive Icon is not set. Android Api 26 push notifications will not be shown if this is not set](#14019--non-adaptive-icon-is-not-set-android-api-26-push-notifications-will-not-be-shown-if-this-is-not-set)
```
Comment (tatianasavinykh) [inline TROUBLESHOOTING-GUIDE.md]: @alexandern17 The link does not work. Also not sure about special char in the headers/TOC

File: TROUBLESHOOTING-GUIDE.md [hunk 4]
```diff
@@ -1,8 +1,20 @@
...
+The Thunderhead SDK for Android Troubleshooting Guide for common implementation issues.
 
-## Integration Issues
+## Table of Contents
+
+- [Integration issues](#integration-issues)
+  * [How to resolve StackOverflow Exception](#how-to-resolve-stackoverflow-exception)
+  * [Resolving conflicts with `android:allowBackup`](#resolving-conflicts-with-android-allowbackup)
+  * [Resolving > The library com.google.firebase:firebase-iid is being requested by various other libraries](#resolving-the-library-comgooglefirebase-firebase-iid-is-being-requested-by-various-other-libraries)
+  * [Resolving: NoSuchMethodError for Base64 class or 15_000: Signpost cannot be used on this platform](#resolving-nosuchmethoderror-for-base64-class-or-15-000-signpost-cannot-be-used-on-this-platform)
```
Comment (tatianasavinykh) [inline TROUBLESHOOTING-GUIDE.md]: @alexandern17 The link does not work. Also not sure about special char in the headers/TOC

File: TROUBLESHOOTING-GUIDE.md (hunk 1 above)
Comment (alexandern17) [reply TROUBLESHOOTING-GUIDE.md]: Interesting... all the links works for me.

Are you testing this page? 

https://github.com/thunderheadone/one-sdk-android/blob/5.0.0-Readme-Updates/TROUBLESHOOTING-GUIDE.md

Comment (skhozin) [reply TROUBLESHOOTING-GUIDE.md]: Does not work for me on this page https://github.com/thunderheadone/one-sdk-android/blob/41c5e997925ce669ce9ca20b48cae4dbae562b62/TROUBLESHOOTING-GUIDE.md#resolving-the-library-comgooglefirebase-firebase-iid-is-being-requested-by-various-other-libraries

Comment (andreipop) [reply TROUBLESHOOTING-GUIDE.md]: @skhozin you should be looking at https://github.com/thunderheadone/one-sdk-android/blob/5.0.0-Readme-Updates/TROUBLESHOOTING-GUIDE.md

File: TROUBLESHOOTING-GUIDE.md (hunk 2 above)
Comment (andreipop) [reply TROUBLESHOOTING-GUIDE.md]: > @alexandern17 The link does not work. Also not sure about special char in the headers/TOC

https://github.com/thunderheadone/one-sdk-android/blob/5.0.0-Readme-Updates/TROUBLESHOOTING-GUIDE.md#resolving--the-library-comgooglefirebasefirebase-iid-is-being-requested-by-various-other-libraries works for me.

File: TROUBLESHOOTING-GUIDE.md (hunk 1 above)
Comment (skhozin) [reply TROUBLESHOOTING-GUIDE.md]: Ok, that one works.

File: TROUBLESHOOTING-GUIDE.md (hunk 2 above)
Comment (skhozin) [reply TROUBLESHOOTING-GUIDE.md]: Works here https://github.com/thunderheadone/one-sdk-android/blob/5.0.0-Readme-Updates/TROUBLESHOOTING-GUIDE.md

File: TROUBLESHOOTING-GUIDE.md (hunk 3 above)
Comment (skhozin) [reply TROUBLESHOOTING-GUIDE.md]: Works here https://github.com/thunderheadone/one-sdk-android/blob/5.0.0-Readme-Updates/TROUBLESHOOTING-GUIDE.md

File: TROUBLESHOOTING-GUIDE.md (hunk 4 above)
Comment (skhozin) [reply TROUBLESHOOTING-GUIDE.md]: Works here https://github.com/thunderheadone/one-sdk-android/blob/5.0.0-Readme-Updates/TROUBLESHOOTING-GUIDE.md

**Pull request merged**